### DISCUSSION TITLE
Blogarchive shortcut is broken

### DISCUSSION THREAD

**Discussion by tschurloIT** [datenstrom/community#627] [Report a bug] [answered]
Hi all!

I downloaded the latest version (Datenstrom Yellow 0.8.17), installed the blog extensions and tried out the various shortcuts.
Unfortunately, I have troubles with the [blogarchive] shortcut.
Using the same blog path as for all the other (working) shortcuts, I get several PHP errors, like: 

`Notice: Undefined variable: entry in C:\xampp\htdocs\sites\yellow\system\extensions\blog.php on line 302`
`Notice: Undefined variable: entry in C:\xampp\htdocs\sites\yellow\system\extensions\blog.php on line 302`
`Notice: Undefined index: 2021-03 in C:\xampp\htdocs\sites\yellow\system\extensions\blog.php on line 303`

There are no corresponding errors in yellow.log

The months listed by this shortcut look fine, but also a superfluous list bullet point is displayed.

Please check and let me know, how to solve this.

Thx a lot!

**Comment by markseuffert** [comment, marked as answer]
Thanks for the detailed information. I updated the [blog extension](https://github.com/annaesvensson/yellow-blog). There was a bug in the function that calculated the months for the `[blogarchive]` shortcut. Should be fixed now, please try it out.

**Reply by tschurloIT** [reply]
Thank you very much for this really quick fix, now it works as it should!

**Reply by markseuffert** [reply]
In the latest version the `[blogarchive]` shortcut has been replaced by:

`[blogyears]` for a list of years
`[blogmonths]` for a list of months

Tested with Datenstrom Yellow 0.8.18.
